### ISSUE TITLE
ACC264.1 (Screen Reader) [fd-form-label][fd-form-message][fd-form-control] Screen reader cannot get the messages

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13222]
Is it a bug ?
 Yes 

In the form, only UI name  is being read but other information such as ‘Error message’ ‘Error icon’ is not being read by the screen reader

Example: 

https://stackblitz.com/edit/cvhgsabu?file=src%2Fapp%2Fform-messaging-state-example.component.html

Expect:

1. Read the required status instead of reading star
2.Read the error status of the input if the  type is error 
3. Read the error messgae 

This is raised by Acessebility Team with high level

Affected Check Point:	[ACC-264](https://wiki.wdf.sap.corp/wiki/display/Acc/ACC-264+Details#ACC264Details-acc2641)

Expected Behavior:

Along with the name and the UI type screen reader should also convey other information (Such as ‘Error message’, ‘Error icon’, etc.….)to the users.

**Comment by droshev** [comment]
@Meggie886 Can you paste the history of the reading from JAWS?

**Comment by Meggie886** [comment]
Hi @droshev  We dont use JAWS,we used the VoiceOver from the Mac OS system.
Here I  cannot paste the screenshot video.I will send you on the teams

Issue: 
1. MessgeTip is not read as a error meesage tip, just the read as the text
2. Required label is read as text without reading the required arritbute
3. Cannot read the error status or the error message in combox and input field


Expected Behavior:
1. Tell the user the it's a meesage tip and it's type
2. Tell the user the field is required if set
3. Tell the user the selected input/combox is in error status and read the error message

**Comment by droshev** [comment]
Can you copy the JAWS history in here?

**Comment by Meggie886** [comment]
Hi @droshev, since we don't use JAWS, here is the screenshot the Accessebility team created in the jira 

![Image](https://github.com/user-attachments/assets/a81394e5-97f0-4ba5-b7e1-91daa3b51d57)

![Image](https://github.com/user-attachments/assets/eee14bf8-a198-4d5e-af0b-52e38c2eb870)

**Comment by droshev** [comment]
It is work in progress. It will come once we adopt https://github.com/SAP/fundamental-styles/pull/6100
